packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -262,7 +262,7 @@ export function RoutingFormResponsesTable({
         routingFormId: selectedRoutingFormId ?? undefined,
         bookingStatus: selectedBookingStatus ?? undefined,
         fieldFilter: selectedRoutingFormFilter ?? undefined,
-        limit: 10,
+        limit: 30,
```

Comment:
The height of each row in this table is pretty small, so we could load 30 items per page, which feels like more performant to users.